.github/workflows/on_target.yml

```diff
@@ -135,107 +92,27 @@ jobs:
               --software-version ${{ inputs.artifact_fw_version }} \
               hello.nrfcloud.com-${{ inputs.artifact_fw_version }}-thingy91x-nrf91.elf
 
-      - name: Run UART tests
-        if: ${{ inputs.run_log_check_tests }}
+      - name: Tests
         working-directory: thingy91x-oob/tests/on_target
         run: |
           mkdir -p results
-          pytest -s -v -m "dut1 and uart" \
-            --junit-xml=results/test-results-uart.xml \
-            tests
-        env:
-          SEGGER: ${{ secrets.SEGGER_DUT_1 }}
-          LOG_FILENAME: oob_uart_test_log
-
-      - name: Run FOTA tests (standard)
-        if: ${{ inputs.run_fota_tests }}
-        working-directory: thingy91x-oob/tests/on_target
-        run: |
-          pytest -s -v -m "dut1 and fota" \
-            --junit-xml=results/test-results-fota.xml \
-            tests
-        env:
-          SEGGER: ${{ secrets.SEGGER_DUT_1 }}
-          IMEI: ${{ secrets.IMEI_DUT_1 }}
-          FINGERPRINT: ${{ secrets.FINGERPRINT_DUT_1 }}
-          LOG_FILENAME: oob_fota_test_log
-
-      - name: Run FOTA tests (FULLMFW)
-        if: ${{ inputs.run_fullmfwfota_test }}
-        working-directory: thingy91x-oob/tests/on_target
-        run: |
-          pytest -s -v -m "dut1 and fullmfw_fota" \
-            --junit-xml=results/test-results-fullmfw-fota.xml \
-            tests
+          pytest ${{ inputs.pytest_marker }} \
+            --junit-xml=results/test-results-dut1.xml \
```

Comment:
arent we running also dut2 and dut_ppk tests in this step?
Also, feel free to change the naming of dut for more appropriate, like maybe dut1->dut_nrf91, dut2->dut_nrf53 or similar

Reply:
Yes, I just changed it to `test-results.xml`. We should look at these names.